src/openrct2/scripting/Plugin.cpp

```diff
@@ -161,4 +162,10 @@ PluginType Plugin::ParsePluginType(const std::string_view& type)
     throw std::invalid_argument("Unknown plugin type.");
 }
 
+void Plugin::CheckForLicence(const std::string_view& licence)
+{
+    if (licence.empty())
+        throw std::invalid_argument("Licence cannot be empty.");
```

Comment:
Frankly, I'm torn on the client enforcing this. I'm fine with reminding devs by means of the definition in `openrct2.d.ts`, but this is of no help to the users.

Reply:
I've changed it to just log an error for now